docs/workflows/ios-specific-workflow-steps/xcode-select.md

```diff
@@ -57,6 +57,20 @@ You can review the documentation for detailed information about the Xcode versio
 
 This step contains some input variable(s). It needs these variable(s) to work. The table below gives explanation for this variable(s).
 
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE5421-xcodeSelectInput.png' />
+
+:::info Manually Xcode Version Change
+
+In addition to the version change method described above, you can also change the Xcode version **manually**. For this, you can **hard-code** the desired Xcode version into the `$AC_XCODE_VERSION` parameter. For example: `15.1`
+
+:::caution Available Xcode Versions
```

Comment:
IMO this nested box usage is not necessary for short infos. Warning can also be written in the same info.
https://capture.dropbox.com/roDX84ZqHJ3czUJ3

For example:
> ...`$AC_XCODE_VERSION` parameter. For example: `15.1`. But if the version you hard-coded is not available on the runner where the build will run...